src/stores/Binding/slices/IncompatibleSchemaChange.ts

```diff
@@ -0,0 +1,43 @@
+import produce from 'immer';
+import { omit } from 'lodash';
+import { NamedSet } from 'zustand/middleware';
+import { BindingState } from '../types';
+
+export interface IncompatibleSchemaChangeDictionary {
+    [uuid: string]: string;
+}
+
+export interface StoreWithIncompatibleSchemaChange {
+    incompatibleSchemaChanges: IncompatibleSchemaChangeDictionary;
+    setSingleIncompatibleSchemaChange: (
+        bindingUUID: string,
+        value: string | undefined
+    ) => void;
+}
+
+export const getInitialIncompatibleSchemaChangeData = (): Pick<
+    StoreWithIncompatibleSchemaChange,
+    'incompatibleSchemaChanges'
+> => ({
+    incompatibleSchemaChanges: {},
+});
+
+export const getStoreWithIncompatibleSchemaChangeSettings = (
+    set: NamedSet<StoreWithIncompatibleSchemaChange>
+): StoreWithIncompatibleSchemaChange => ({
+    ...getInitialIncompatibleSchemaChangeData(),
+
+    setSingleIncompatibleSchemaChange: (bindingUUID, value) => {
+        set(
+            produce((state: BindingState) => {
+                if (value) {
+                    state.incompatibleSchemaChanges[bindingUUID] = value;
+                } else {
+                    omit(state.incompatibleSchemaChanges, bindingUUID);
+                }
```

Comment:
Please move this over to `workflow-utils` 

I know it isn't shared but I think we need to start getting all our "schema updates" into a single location so we can start building out a solid reusable util for schema property updates